packages/platforms/react-native/lib/span-attributes-source.ts

```diff
@@ -1,14 +1,25 @@
 import { type SpanAttributesSource } from '@bugsnag/core-performance'
-import { type ReactNativeConfiguration } from './config'
+import NetInfo from '@react-native-community/netinfo'
 import { type AppStateStatic } from 'react-native'
+import { type ReactNativeConfiguration } from './config'
+import getNetworkConnectionType from './get-network-connection-type'
 
 export function createSpanAttributesSource (appState: AppStateStatic) {
+  let connectionType = 'unknown' // TODO: do we need to set an initial value or will the event listener fire in the first instance?
```

Comment:
I think this TODO can be removed because NetInfo should call the event listener with the current state on subscribe